apps/src/labs/README.md

```diff
@@ -0,0 +1,21 @@
+# Lab2
```

Comment:
this readme is not complete yet, but I wanted to get something started to establish the Lab2 name

Reply:
Thanks for adding this!